notebooks/concepts/generate_mosviz_photometry.ipynb

```diff
@@ -0,0 +1,837 @@
+  {
+   "cell_type": "markdown",
+   "id": "5f2ade07-e354-499d-b27d-fd383e3353e3",
+   "metadata": {},
+   "source": [
+    "At this point, we are ready to ready to create and save cutouts using the source locations found earlier.\n",
+    "\n",
+    "_Note: Depending on the value of `catalog_size`, the following cell can produce a lot of output. Right-click the cell and select \"Disable Scrolling for Outputs\" to expand it or \"Enable Scrolling for Outputs\" to condense it._"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "edf1b0a4-1132-42e8-b6fd-12436f2355b5",
+   "metadata": {
+    "scrolled": true,
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "# define parameters for cutout loop\n",
+    "all_cutouts = []\n",
+    "catalog_size = 33 # how many sources to include in the cutout list\n",
+    "cutout_length = 100 # the maximum length of a cutout in pixels\n",
+    "downsample_factor = 4 # scale sources down to proper size for the new image\n",
+    "\n",
+    "for i in sources['label']:\n",
+    "    # save and plot the new cutout\n",
+    "    segm_source = segm_deblend.segments[i - 1]\n",
+    "    # (sources[\"label\"] is 1-indexed, so subtract 1 for matching segm index)\n",
+    "\n",
+    "    # implement the downsample\n",
+    "    cutout = segm_source.make_cutout(image_data, masked_array=True)\n",
+    "    cutout = block_reduce(cutout, downsample_factor) / downsample_factor**2\n",
+    "    \n",
+    "    plt.imshow(cutout, vmin=0, vmax=image_data.std()*3, **imshow_params)\n",
+    "    plt.show()\n",
+    "    \n",
+    "    all_cutouts.append(cutout)\n",
+    "    if len(all_cutouts) == catalog_size:\n",
+    "        break"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ace709b6-aba7-4ec1-8733-99acdebc4dda",
+   "metadata": {},
+   "source": [
+    "___"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "87691179-7fd9-497c-9260-8f24c8b31d41",
+   "metadata": {},
+   "source": [
+    "### _[An alternate, catalog file-based cutout generation process]_\n",
+    "\n",
+    "It is also possible to build a list of cutouts from an already-existing source catalog; this section serves as a reference for how it is done. Its cells are not necessary for the completion of the notebook and do not run due to the `%%script false --no-raise-error` statements at the top of each one.\n",
+    "\n",
+    "Depending on the catalog, doing it this way can give you more detailed information about the resulting sources, but the process is less generalizable from one telescope's/instrument's/detector's images to another's. This path may also be less desirable for synthetic image creation because the cutouts will be square patches and thus look a little more, well, synthetic."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "b2f3f95b-2d95-413d-9080-79651e8f03ac",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%%script false --no-raise-error\n",
```

Comment:
Instead of inserting `%%script`, why not convert the cell to "raw NBconvert" type instead? That way, when someone wants to run it, they convert it back to actual code cell.

Reply:
Are there issues with `%%script` to be concerned about? I was hoping to preserve those cells' syntax highlighting while also not having them run by default.

Reply:
It is fine if you want to keep them, especially since this is an exploratory notebook.